static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -367,16 +357,12 @@ export function DisasterEventMapTile(
           placeInfo.enclosedPlaceType
         )
       : Promise.resolve(mapChartData.geoJson);
-    const dateRange: [string, string] =
-      selectedDate in dateRanges.current
-        ? dateRanges.current[selectedDate]
-        : [selectedDate, selectedDate];
     const disasterEventDataPromise = fetchDisasterEventPoints(
-      Object.values(eventTypeSpec),
-      placeInfo.selectedPlace.dcid,
-      dateRange,
-      severityFilters,
-      useCache
+      dataOptions.eventTypeSpecs,
+      dataOptions.place,
+      dataOptions.selectedDate,
+      dataOptions.severityFilters,
+      dataOptions.useCache
```

Comment:
should we pass in the dataOptions object instead?

Reply:
that makes sense, updated!